generators/testing/templates/index.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
```

Comment:
We're not using this file at all. It can be removed/deleted.

Reply:
Deleted. Thanks.